streaming/java/streaming-runtime/src/test/java/org/ray/streaming/runtime/python/MsgPackSerializerTest.java

```diff
@@ -0,0 +1,30 @@
+package org.ray.streaming.runtime.python;
+
+import static org.testng.Assert.*;
+
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.testng.annotations.Test;
+
+@SuppressWarnings("unchecked")
+public class MsgPackSerializerTest {
+
```

Comment:
Better use a meaningful test case and make sure the core logic of the MsgPackSerializer has been tested.

Reply:
done